vital/types/detected_object.cxx

```diff
@@ -78,16 +94,39 @@ bounding_box_d
 detected_object
 ::bounding_box() const
 {
-  return *m_bounding_box;
+  return m_bounding_box;
 }
 
-
 // ------------------------------------------------------------------
 void
 detected_object
 ::set_bounding_box( const bounding_box_d& bbox )
 {
-  m_bounding_box = std::make_shared< bounding_box_d >( bbox );
+  m_bounding_box = bbox ;
+}
+
+// ------------------------------------------------------------------
+geo_point_sptr&
+detected_object
+::location()
+{
+  return m_location;
+}
+
+// ------------------------------------------------------------------
+geo_point_cptr
+detected_object
+::location() const
+{
+  return m_location;
+}
+
+// ------------------------------------------------------------------
+void
+detected_object
+::set_location(const geo_point& pt)
```

Comment:
Style, again...

(FYI, I probably wouldn't change it here because this file is "consistently wrong", but the unofficial guideline is to use `T const` rather than `const T`.)